README.md

```diff
@@ -61,14 +61,14 @@ Spanner Autoscaler can be installed using [KPT](https://kpt.dev/installation/) b
    ```
    > :information_source: **TIP:** Instead of `kpt`, you can also use `kubectl` directly to apply the resources with
    >   ```console
-   >   $ kustomize build spanner-autoscaler/default | kubectl apply -f -
+   >   $ kustomize build config/default | kubectl apply -f -
```

Comment:
The original `kpt` command for getting the package, downloads the contents of `spanner-autoscaler/config` to the `spanner-atusocaler` directory (from `$ kpt pkg get https://github.com/mercari/spanner-autoscaler/config spanner-autoscaler` command mentioned above).
I think it is confusing because of the same name for the kpt package as well the repository. Probably the kpt package repository name should be something different to avoid this confustion for users.

Reply:
I have updated the PR to change the `kpt` package directory name from `spanner-autoscaler` to ` spanner-autoscaler-pkg` to avoid the confusion. And have also added URL link for direct `kustomize` apply instead of local directory.